test/algo/compression/pruning/test_concrete_trace_mmdetection.py

```diff
@@ -0,0 +1,287 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import pytest
+
+import os
+import traceback
+import torch
+import mmcv
+import mmcv.cnn as mmcv_cnn
+import mmdet.core as mmdet_core
+from mmdet.apis import init_detector
+from mmcv.parallel import collate
+from mmdet.datasets import replace_ImageToTensor
+from mmdet.datasets.pipelines import Compose
+from nni.common.concrete_trace_utils import concrete_trace, ConcreteTracer
+
+assert 'MMDET_DIR' in os.environ, 'please set env variable `MMDET_DIR` to your mmdetection folder!'
```

Comment:
Suggest `pytest.skip` if MMDET is not found. So that the test collection doesn't fail simply because they didn't set MMDET_DIR.